packages/vuelidate/src/core.js

```diff
@@ -91,8 +91,9 @@ function normalizeValidatorResponse (result) {
  * @param {Ref<*>} model
  * @return {Ref<Boolean>}
  */
-function createComputedResult (rule, model) {
+function createComputedResult (rule, model, $dirty) {
```

Comment:
This is part of the lazy by default functionality, right?

Reply:
Yup. Sorry, I mixed the two PRs.